src/components/icon/index.tsx

```diff
@@ -25,6 +25,14 @@ export type IconProps = Omit<ImageProps, 'source'> &
      * the icon size
      */
     size?: number;
+    /**
+     * the icon height
+     */
+    height?: number | string;
```

Comment:
About the `string` type, when I pass the string I get this error:
```
JSON value of type NSString cannot be converted to a YGValue.
```

To get over this error the user should pass `value%` as the `width` or `height` value and this is not the result we want.

Let's make it only a `number` type.

Reply:
@adids1221 what string did you use?